app/javascript/psulib_blacklight/javascripts/libraries_locations.json

```diff
@@ -1,716 +1,716 @@
+    "NEWS-PAT": "Paterno - 1st Floor - Newspaper Display Area",
+    "NONBOOK-MA": "Mont Alto - Non-Book Collection, Ask at Desk",
+    "NONCIRC-EG": "Engineering, 325 Hammond Bldg. - Non Circulating",
+    "NONPRINTGV": "Great Valley - Nonprint Collection, Ask at Desk",
+    "NONPRTR-GV": "Great Valley - Nonprint Reference, Ask at Desk",
+    "OCCULT": "Rare Books & Mss, 1st Floor Paterno, Occult Collection",
+    "OFFSITEADN": "Dickinson Law (Carlisle) - Offsite A Collection",
+    "OFFSITE-BU": "Business Entrepreneurship Cooperative Collection - Offsite",
+    "OFFSITE-DN": "Dickinson Law (Carlisle) - Offsite Collection",
+    "OFFSITE-NK": "New Kensington - Entrepreneurship Collection at the Corner",
+    "OHARA-SL": "Schuylkill - O'Hara Collection - In-Library Use",
+    "OHARA": "Rare Books & Mss, 1st Floor Paterno, John O'Hara Coll.",
+    "ONHOLD-AA": "On hold at Penn State Altoona",
+    "ONHOLD-AB": "On hold at Penn State Abington",
+    "ONHOLD-AC": "On hold at Architecture & Landscape Arch, Stuckeman Bldg",
+    "ONHOLD-BD": "On hold at Penn State Erie",
+    "ONHOLD-BK": "On hold at Penn State Berks",
+    "ONHOLD-BR": "On hold at Penn State Beaver",
+    "ONHOLD-DE": "On hold at Penn State Brandywine",
+    "ONHOLD-DN": "On hold at Dickinson Law (Carlisle)",
+    "ONHOLD-DP": "On hold at Penn State Law (UP)",
+    "ONHOLD-DS": "On hold at Penn State Dubois",
+    "ONHOLD-EG": "On hold at Engineering Library, 325 Hammond Bldg.",
+    "ONHOLD-EM": "On hold at Earth & Mineral Sci Library, 105 Deike Bldg.",
+    "ONHOLD-FE": "On hold at Penn State Fayette",
+    "ONHOLD-GV": "On hold at Penn State Great Valley",
+    "ONHOLD-HB": "On hold at Penn State Harrisburg",
+    "ONHOLD-HN": "On hold at Penn State Hazleton",
+    "ONHOLD-HY": "On hold at Penn State Hershey (College of Medicine)",
+    "ONHOLD-LV": "On hold at Penn State Lehigh Valley",
+    "ONHOLD-MA": "On hold at Penn State Mont Alto",
+    "ONHOLD-MD": "On hold at College of Medicine (UPP)",
+    "ONHOLD-MK": "On hold at Penn State Greater Allegheny",
+    "ONHOLD-MT": "On hold at Mathematics Library, 109 McAllister Bldg.",
+    "ONHOLD-NK": "On hold at Penn State New Kensington",
+    "ONHOLD-PA": "On hold at Pattee/Paterno",
+    "ONHOLD-PM": "On hold at Physical & Mathematical Sci. Lib., 201 Davey Lab",
+    "ONHOLD-PO": "On hold at Pollock Laptop Library",
+    "ONHOLD-SL": "On hold at Penn State Schuylkill",
+    "ONHOLD-SV": "On hold at Penn State Shenango",
+    "ONHOLD-WB": "On hold at Penn State Wilkes-Barre",
+    "ONHOLD-WD": "On hold for World Campus",
+    "ONHOLD-WS": "On hold at Penn State Scranton",
+    "ONHOLD-YK": "On hold at Penn State York",
+    "ONLINE-DSL": "Online Content (Law Schools)",
+    "ONLINE": "Online Content",
+    "ON-ORDER": "Being acquired by the library",
+    "OPAQUE-NML": "2nd Floor Paterno - Microprint",
+    "OPENRES-AC": "Architecture & Landscape Arch, Stuckeman Bldg â€“ Service Desk",
```

Comment:
@ruthtillman one more weird character here in this line

Reply:
```suggestion
    "OPENRES-AC": "Architecture & Landscape Arch, Stuckeman Bldg – Service Desk",
```

Reply:
@ruthtillman I just updated it with a `–`, hope that was correct

Reply:
yep thank you.